broker/l2tp_broker.cfg.example

```diff
@@ -21,6 +21,9 @@ namespace=default
 connection_rate_limit=10
 ; Set PMTU to a fixed value.  Use 0 for automatic PMTU discovery.
 pmtu=0
+; Whether this server runs a new kernel (4.13 and various stable series; in particular, 4.9.36) and
```

Comment:
Could we perform some autodetection to determine this?

Reply:
I guess we could -- but isn't that too fragile? I don't even have complete data which other kernels the "bad" patch was backported to.

Reply:
However, one thing we *could* do is detect the EEXIST on l2tpv3 session creation, and treat that as a hint that we need unique sessions IDs.  That would mean one client has a failed connection attempt, and old clients take 30s to re-try, but the next time around they would see usage 0xFFFF.

Would you prefer that?